src/main/kotlin/model/lotto/LottoTicket.kt

```diff
@@ -1,41 +1,13 @@
 package model.lotto
 
-enum class TicketType {
-    Manual,
-    Auto,
-}
-
-data class LottoTicket(
-    val numbers: LottoNumbers,
-    val type: TicketType,
-) {
+class LottoTicket(val games: List<LottoGame>) {
```

Comment:
```suggestion
class LottoTicket private constructor(val games: List<LottoGame>)
```

찾아보니, 정적 팩토리 메서드가 있다면, `private constructor` 를 붙여서 생성자 접근제어를 막을 수도 있는 것 같아요